gms/api/src/main/pegasus/com/linkedin/glossary/GlossaryTermKey.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.GlossaryNodeUrn
+
+/**
+ * Key for GlossaryTerm resource
+ */
+record GlossaryTermKey {
+
+  @validate.strlen = {
+    "max" : 20,
+    "min" : 1
+  }
+  name: string
+
+  /**
+   * Parent business nodes(in sequence) to this business term.
+   */
+  glossaryNodePath: string
```

Comment:
why is the TermKey is structured differently than the TermUrn?

Reply:
will be removing `glossaryNodePath` [same as above]

Reply:
done